spec/factories/courses.rb

```diff
@@ -27,12 +27,46 @@
     name { Faker::ProgrammingLanguage.name }
     qualification { 1 }
     association(:provider)
-
+    profpost_flag { %i[recommendation_for_qts professional postgraduate professional_postgraduate].sample }
```

Comment:
Sensible one will be any but recommendation_for_qts

And have the qts pgce qualification as that don't touch further education and pgde_course